examples/tweet_generator/tweet_generator.py

```diff
@@ -0,0 +1,153 @@
+import re
+import textwrap
+from datetime import datetime
+from PIL import Image, ImageFont, ImageDraw, ImageOps
+from pincer import command, Client, Descripted
+from pincer.objects import Message, Embed, MessageContext
+
+
+class Bot(Client):
+    @Client.event
+    async def on_ready(self):
+        print(
+            f"Started client on {self.bot}\n"
+            "Registered commands: " + ", ".join(self.chat_commands)
+        )
+
+    @command(
+        name="twitter",
+        description="to create fake tweets",
+    )
+    async def twitter(
+        self, ctx: MessageContext, content: Descripted[str, "..."]
+    ):
+        await ctx.interaction.ack()
+
+        message = content
+        for text_match, user_id in re.findall(
+            re.compile(r"(<@!(\d+)>)"), message
+        ):
+            message = message.replace(
+                text_match, "@%s" % await self.get_user(user_id)
+            )
+
+        if len(message) > 280:
+            return "A tweet can be at maximum 280 characters long"
+
+        # wrap the message to be multi-line
+        message = textwrap.wrap(message, 38)
+
+        # download the profile picture and convert it into Image object
+        avatar = (await ctx.author.user.get_avatar()).resize((128, 128))
+
+        # modify profile picture to be circular
+        mask = Image.new("L", (128, 128), 0)
+        draw = ImageDraw.Draw(mask)
+        draw.ellipse((0, 0) + (128, 128), fill=255)
+        avatar = ImageOps.fit(avatar, mask.size, centering=(0.5, 0.5))
+        avatar.putalpha(mask)
+
+        # create the tweet by pasting the profile picture into a white image
+        tweet = trans_paste(
+            avatar,
+            # background
+            Image.new("RGBA", (800, 250 + 50 * len(message)), (255, 255, 255)),
+            box=(15, 15),
+        )
+
+        # add the fonts
+        font = ImageFont.truetype("Segoe_UI.ttf", 40)
+        font_small = ImageFont.truetype("Segoe_UI.ttf", 30)
+        font_bold = ImageFont.truetype("Segoe_UI_Bold.ttf", 40)
+
+        # write the name and username on the Image
+        draw = ImageDraw.Draw(tweet)
+        draw.text(
+            (180, 20), str(ctx.author.user), fill=(0, 0, 0), font=font_bold
+        )
+        draw.text(
+            (180, 70),
+            "@" + ctx.author.user.username,
+            fill=(120, 120, 120),
+            font=font,
+        )
+
+        # write the content of the tweet on the Image
+        message = "\n".join(message).split(" ")
+        result = []
+
+        # generate a dict to set were the text need to be in different color.
+        # for example, if a word starts with '@' it will be write in blue.
+        # example:
+        #   [
+        #       {'color': (0, 0, 0), 'text': 'hello world '},
+        #       {'color': (0, 154, 234), 'text': '@drawbu'}
+        #   ]
+        for word in message:
+            for index, text in enumerate(word.split("\n")):
```

Comment:
There's a separate method for splitting newlines
```suggestion
            for index, text in enumerate(word.splitlines()):
```